### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by AshutoshM29**
Just some small refactors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <div class="nav-section">
```
Comment (JuvitaSaini09) [inline index.html]: u could have used nav element

File: styles.css [hunk 2]
```diff
@@ -261,7 +261,7 @@ iframe {
   justify-content: flex-start;
 }
 
-@media screen and ( max-width:600px){
```
Comment (JuvitaSaini09) [inline styles.css:264]: u r missing @media only screen

**Review by JuvitaSaini09 — APPROVED** [review]

**Review by piyushsh25 — APPROVED** [review]
Looks good to me..... Small suggestion: you can use px value where size is less than 1rem.You can merge.

**Comment by AshutoshM29** [comment]
> Looks good to me..... Small suggestion: you can use px value where size is less than 1rem.You can merge.

Is it necessary to use px, cuz Vikram said it's good if we use rem!

**Pull request merged**